### PR TITLE
test collaberation

### PR DESCRIPTION
**Pull request by tonyt50**
pull request for testing

### CODE REVIEW

File: testfunctions.js [hunk 1]
```diff
@@ -43,9 +43,9 @@ function test001(inputstring) {
       } else {
         maxoffset = offsets[offsetsindex] - 1;
       }
-//      possiblelengths["" + minoffset + maxoffset] = {
+      possiblelengths["" + minoffset + maxoffset] = {
```
Comment (jedster1111) [inline testfunctions.js]: Wouldn't do that

File: testfunctions.js [hunk 2]
```diff
@@ -43,9 +43,9 @@ function test001(inputstring) {
...
       // TODO here check if valid, as dont check current multipleletterskey
-      possiblelengths[possiblelengthskey++] = {
+//      possiblelengths[possiblelengthskey++] = {
```
Comment (jedster1111) [inline testfunctions.js]: But I guess this makes sense

**Review by jedster1111 — CHANGES_REQUESTED** [review]
Think you need some more comments

File: testfunctions.js (hunk 2 above)
Comment (tonyt50) [reply testfunctions.js]: Fixed in fbf02ad

File: testfunctions.js (hunk 1 above)
Comment (jedster1111) [reply testfunctions.js]: Looks good now

File: testfunctions.js (hunk 2 above)
Comment (jedster1111) [reply testfunctions.js]: Yeah that looks good

**Comment by jedster1111** [comment]
Looks good to me, go for the merge

**Pull request merged**